liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java

```diff
@@ -76,14 +78,20 @@ public synchronized void updateSettings(Object initializationOptionsSettings) {
         // TODO: else update settings
     }
 
+    @Override
+    public void connect(LanguageClient client) {
+        this.languageClient = client;
+    }
+
     @Override
     public CompletableFuture<Object> shutdown() {
-        // TODO Auto-generated method stub
-        return null;
+        super.shutdownServer();
+        return CompletableFuture.completedFuture(new Object());
```

Comment:
The contents of the shutdown and exit method look pretty different to me from before. This is why I thought behavior might be different?

Reply:
Ah I see. The updated methods are just to ensure the language server instances from the tests are closed out.